www/index.html

```diff
@@ -584,7 +584,7 @@ <h3>Compatibility settings</h3>
                                     <label title="This mode requires that the browser or framework be capable of installing a Service Worker. It works by intercepting the browser or framework's Fetch calls and supplying the requested content from the ZIM.">
                                         <input type="radio" name="contentInjectionMode" value="serviceworker"
                                             id="serviceworkerModeRadio">
-                                        <strong>ServiceWorker</strong> (default, supports archives with dynamic content)
+                                        <strong>Service Worker</strong> (default, supports archives with dynamic content)
```

Comment:
Again, this is short for `ServiceWorker Mode` (it's clearly being used as a description of the mode), so it should remain as `ServiceWorker`.